spec/rmagick/image_list/new_image_spec.rb

```diff
@@ -12,5 +12,7 @@
     image_list.new_image(20, 20) { self.background_color = 'red' }
     expect(image_list.length).to eq(3)
     expect(image_list.scene).to eq(2)
+
+    expect { image_list.new_image(20, 20, nil) }.not_to raise_error
```

Comment:
@Watson1978 why do we want to allow passing `nil` here? Should it be the 2 argument version from above?

Reply:
Currently, the default value of `fill` argument is set to false. but it is not used correctly
if pass nil to indicate that the fill object has been omitted, this method raises an exception.

Reply:
I needed to fix to write a proper explanation in https://github.com/rmagick/rmagick/pull/1123